2Dgame/Assets/Scripts/CameraScript.cs

```diff
@@ -23,12 +23,46 @@ public class CameraScript : MonoBehaviour {
 	 * Uses linear interpolation to make this switch in speed feel smooth.
 	 */
 	void ScrollUp() {
-		transform.position = new Vector3 (transform.position.x, transform.position.y + scrollSpeed*Time.deltaTime, transform.position.z);
+        StandardSpeed();
 
-		if (target.position.y - transform.position.y > 5f) {
-			scrollSpeed = Mathf.Lerp (scrollSpeed, target.velocity.y, 0.1f);
-		} else {
-			scrollSpeed = Mathf.Lerp(scrollSpeed, standardSpeed, 2f);
-		}
+        if (target.position.y - transform.position.y > 3f)
+        {
+            scrollSpeed = Mathf.Lerp(scrollSpeed, target.velocity.y, LerpSpeed());
+        }
+        else
+        {
+            scrollSpeed = Mathf.Lerp(scrollSpeed, standardSpeed, LerpSpeed());
+        }
+
+        transform.position = new Vector3 (transform.position.x, transform.position.y + scrollSpeed * Time.deltaTime, transform.position.z);
 	}
+
+    /**
+     * Scrolls up in different speeds, depending on score.
+     * To make the start easier.
+     */
+    void StandardSpeed()
+    {
+        if (transform.position.y < 200)
+        {
+            standardSpeed = 2f;
```

Comment:
It might be just me who's played this game a lot but this speed felt very slow. Maybe change this to 3f? What do you think? Like I would have to jump on a platform several times to even get this speed